codes/classical/properties/Low Density Parity Check (LDPC)

```diff
@@ -0,0 +1,33 @@
+
+code_id: LDPC
+
+name: 'Asymptotically Good LDPC Convolutional Codes Based on Protographs'
+introduced: '\cite{doi:10.1070/RM1997v052n06ABEH002155}'
+
+
+Description:
+- 'LDPC or Low Density Parity Check codes: A (N, J, K) binary code which its parity check matrix, H, has N columns, J ones in each column, K ones in each row and zeros elsewhere. This code can be built from a sparse bipartite graph (reffered to asTanner graph). (details in notes)'
+- 'Convolutional (Cyclic) LDPC codes: Given a parity check matrix H, we can construct a (M, J, K) code with rate b/c code which its parity check matrix is semi infinite, has J ones in each column, K ones in each row starting from the M(c-b)th column and zero elsewhere. This code can be built by copying (reffered to as folding or lifting) a sparse bipartite graph infinite times and look at it as a huge bipartite graph. (details in notes)'
```

Comment:
Again, I am not sure which codes you are defining. Should convolutional codes appear here?